### DISCUSSION TITLE
1월 9일 영어 회화 연습

### DISCUSSION THREAD

**Discussion by giscus** [Cyma-s/Cyma-s.github.io#102] [General]
# 1월 9일 영어 회화 연습

오늘의 키워드 어떻게 하면 말을 잘 정리해서 말할 수 있을까? 한국어 나는 말을 정리해서 말하는 것을 잘 못하는 편이다. 간단한 문장도 머릿속에서 뒤죽박죽이 되곤 한다. 영어로 말하는 것은 말할 것도 없다. 한국어로 생각하는 것도 어려운데 영어로 말하는 게 쉬울 리 없다. 특히 내가 떠올린 문장을 기억하는 것이 내게 가장 어려운 부분이다. 나는 단기 기억…

https://vero.wiki/english-speaking-2024-01-09/

**Comment by somsom13** [comment]
> 나는 말을 정리해서 말하는 것을 잘 못하는 편이다. 

....? ??? ?????

> 나는 단기 기억력이 좋지 않다

..???? ?????
하지만 제가 기억하는 베로는 말도 잘하고 기억력도 좋은 사람인걸요..

**Reply by Cyma-s** [reply]
ㅋㅋㅋㅋㅋㅋㅋㅋㅋ 전 말하는 감자에요.... 🥹🥔
